### DISCUSSION TITLE
CTM?

### DISCUSSION THREAD

**Discussion by bananarceus** [IrisShaders/Iris#2194] [General]
Hi! Is there CTM for Iris/Sodium?

I tested out complementary shaders on the Iris/Sodium build for the Lunar Client, and they work better there than on Optifine, and I can't find info about this elsewhere, but is there already a connected textures mod built into Iris or is it not a thing?

If there is one, what do I do to my texture pack to make it work, and if there isn't, is this in development and/or is there a way to get it to work via additional mods or anything, and will I be able to load that on the Lunar Client? Thanks in advance!

**Comment by Player005** [comment]
Hi there,
Iris is a shaders mod only, there are separate mods for each optifine feature, for connected textures there is the [continuity mod](https://modrinth.com/mod/continuity), which is compatible with the optifine format so you don't need to modify your texture pack in any way. However, as far as I'm aware, lunar client does not allow you to add your own mods to it, so you'll have to use a normal fabric install instead of lunar client. You can also use a modpack like [additive](https://modrinth.com/modpack/additive), which has all the performance mods and optifine replacement mods and you can add your own favourite mods to it if you want.

**Reply by bananarceus** [reply]
Thanks lots!! I'll be looking into those. It'd be cool if there was a launcher that had that stuff ngl, I don't like the whole telemetry thing oof

will get continuity or additive!
